### DISCUSSION TITLE
No publishable resources for tag [showsql-config].

### DISCUSSION THREAD

**Discussion by laTruffe79** [dietercoopman/laravel-showsql#8] [Q&A]
Hello, I just installed the package but the command in order tu publish the config file doesn't work and give me the message No publishable resources for tag [showsql-config].
My stack is php8.0-fpm laravel 8.83.9
thanks.

**Comment by dietercoopman** [comment]
Hi,

I've just tried it and it works here on php8.0.17 , did you run  `php artisan vendor:publish --tag=showsql-config`

Kind regards

**Reply by laTruffe79** [reply]
yes sure and the response is : No publishable resources for tag [showsql-config].

**Comment by laTruffe79** [comment]
![image](https://user-images.githubusercontent.com/7983536/164451149-a59e1bf4-447f-4fb2-bf94-0ec196188029.png)

**Reply by dietercoopman** [reply]
Hi , this was indeed a bug and it is fixed in version 1.3.3 , please run a composer update.

Thx 🤝

**Reply by laTruffe79** [reply]
Thanks

**Comment by laTruffe79** [comment]
I tried to update to 1.3.3 but that gives me this error : composer.json requires dietercoopman/laravel-showsql 1.3.3, found dietercoopman/laravel-showsql[dev-main, v1.0, ..., v1.3.1] but it does not match the constraint.

**Reply by dietercoopman** [reply]
Could you please retry , there is a problem with auto-deploy to packagist. Should have been solved now.

Regards Dieter

**Reply by laTruffe79** [reply]
It's OK now, thank you very much !
